open-metadata-implementation/framework-services/ocf-metadata-management/ocf-metadata-api/src/main/java/org/odpi/openmetadata/frameworkservices/ocf/metadatamanagement/api/ConnectorFactoryInterface.java

```diff
@@ -8,6 +8,7 @@
 import org.odpi.openmetadata.frameworks.connectors.ffdc.InvalidParameterException;
 import org.odpi.openmetadata.frameworks.connectors.ffdc.PropertyServerException;
 import org.odpi.openmetadata.frameworks.connectors.ffdc.UserNotAuthorizedException;
+import org.odpi.openmetadata.frameworks.connectors.properties.beans.Asset;
```

Comment:
This interface is a factory that returns connectors (as per its name). It does not seem right to add a method to return an asset here.